syntax = "proto3";

package example;

service Example {
    rpc Ping(PingRequest) returns (PongResponse);
    rpc PingStream(stream PingRequest) returns (stream PongResponse);
}

message PingRequest {
    string message = 1;
}

message PongResponse {
    string message = 1;
}